src/icon/styled-components.js

```diff
@@ -14,8 +14,6 @@ export function getSvgStyles({$theme, $size, $color}: StyledComponentParamsT) {
       $size = $theme.sizing[$size];
     } else if (typeof $size === 'number') {
       $size = `${$size}px`;
-    } else {
-      $size = $theme.sizing.scale600;
```

Comment:
The custom size string still won't work I think.

Reply:
discussed this offline